plugins/epos/plugin.py

```diff
@@ -168,6 +168,136 @@ def rda_f4_01m(self):
+        to the digital object. It is (i) about whether there are restrictions to access the data (i.e.
+        access to the data may be open, restricted or closed), (ii) the actions to be taken by a
+        person who is interested to access the data, in particular when the data has not been
+        published on the Web and (iii) specifications that the resources are available through
+        eduGAIN7 or through specialised solutions such as proposed for EPOS.
+        Technical proposal: Resolve the identifier
+        Parameters
+        ----------
+        item_id : str
+            Digital Object identifier, which can be a generic one (DOI, PID), or an internal (e.g. an
+            identifier from the repo)
+        Returns
+        -------
+        points
+            A number between 0 and 100 to indicate how well this indicator is supported
+        msg
+            Message with the results or recommendations to improve this indicator
+        """
+        # 1 - Check metadata record for access info
+        msg = (
+            "%s: "
+            % _(
+                "No access information can be found in the metadata. Please, add information to the following term(s): %s"
+            )
+            % self.terms_access
+        )
+        points = 0
+        md_term_list = pd.DataFrame(self.terms_access, columns=["term", "qualifier"])
+        md_term_list = ut.check_metadata_terms(self.metadata, md_term_list)
+        if sum(md_term_list["found"]) > 0:
+            for index, elem in md_term_list.iterrows():
+                if elem["found"] == 1:
+                    msg = _(
+                        "| Metadata: %s.%s: ... %s"
+                        % (
+                            elem["term"],
+                            elem["qualifier"],
+                            self.metadata.loc[
+                                self.metadata["element"] == elem["term"]
+                            ].loc[self.metadata["qualifier"] == elem["qualifier"]],
+                        )
+                    )
+                    points = 100
+
+        if points == 0:
+            md_term_list = pd.DataFrame(self.doi, columns=["term", "qualifier"])
+            md_term_list = ut.check_metadata_terms(self.metadata, md_term_list)
+            if sum(md_term_list["found"]) > 0:
+                for index, elem in md_term_list.iterrows():
+                    if elem["found"] == 1:
+                        msg = _(
+                            "| Metadata: %s.%s: ... %s"
+                            % (
+                                elem["term"],
+                                elem["qualifier"],
+                                self.metadata.loc[
+                                    self.metadata["element"] == elem["term"]
+                                ].loc[self.metadata["qualifier"] == elem["qualifier"]],
+                            )
+                        )
+                        points = 100
```

Comment:
Repeating blocks of code should be avoided. As I understand this can be sorted out by adding the DOI to the `terms_access` in config.ini, such as:

```
 # Metadata terms that defines accessibility
 terms_access = [['downloadURL',''], ['DOI','']]
```

Reply:
Added and works nicely

Reply:
great! we can do the same with `license` so that we have all the metadata elements related to accessibility in `terms_access`

Reply:
The problem is that terms_acces is used for more tests than this test so adding the license to this category could be troublesome